diff --git a/dist/ts-morph-common.js b/dist/ts-morph-common.js
index 5026b0032bfaaa7b291b4750df5abffc7c7edc17..b7b3162501132fd1c0607f2701b28d14ddcc144c 100644
--- a/dist/ts-morph-common.js
+++ b/dist/ts-morph-common.js
@@ -4,7 +4,7 @@ Object.defineProperty(exports, '__esModule', { value: true });
 
 var ts = require('./typescript');
 var minimatch = require('minimatch');
-var fastGlob = require('fast-glob');
+var tinyglobby = require('tinyglobby');
 var fs$1 = require('fs');
 var mkdirp = require('mkdirp');
 var os = require('os');
@@ -32,7 +32,6 @@ function _interopNamespace(e) {
 
 var ts__namespace = /*#__PURE__*/_interopNamespace(ts);
 var minimatch__default = /*#__PURE__*/_interopDefaultLegacy(minimatch);
-var fastGlob__default = /*#__PURE__*/_interopDefaultLegacy(fastGlob);
 var fs__namespace = /*#__PURE__*/_interopNamespace(fs$1);
 var mkdirp__default = /*#__PURE__*/_interopDefaultLegacy(mkdirp);
 var os__namespace = /*#__PURE__*/_interopNamespace(os);
@@ -849,7 +848,7 @@ class BrowserRuntime {
         return "\n";
     }
     getPathMatchesPattern(path, pattern) {
-        return minimatch__default["default"](path, pattern);
+        return minimatch.minimatch(path, pattern);
     }
 }
 class BrowserRuntimePath {
@@ -995,7 +994,7 @@ class NodeRuntime {
         return os__namespace.EOL;
     }
     getPathMatchesPattern(path, pattern) {
-        return minimatch__default["default"](path, pattern);
+        return minimatch.minimatch(path, pattern);
     }
 }
 class NodeRuntimePath {
@@ -1131,15 +1130,17 @@ class NodeRuntimeFileSystem {
         return path__namespace.resolve();
     }
     glob(patterns) {
-        return fastGlob__default["default"](patterns, {
+        return tinyglobby.glob(patterns, {
             cwd: this.getCurrentDirectory(),
             absolute: true,
+            expandDirectories: false,
         });
     }
     globSync(patterns) {
-        return fastGlob__default["default"].sync(patterns, {
+        return tinyglobby.globSync(patterns, {
             cwd: this.getCurrentDirectory(),
             absolute: true,
+            expandDirectories: false,
         });
     }
     isCaseSensitive() {
